iceberg/iceberg-rest-server/src/main/java/org/apache/gravitino/iceberg/service/rest/IcebergViewOperations.java

```diff
@@ -139,11 +185,49 @@ public Response viewExists(
       @PathParam("prefix") String prefix,
       @PathParam("namespace") String namespace,
       @PathParam("view") String view) {
-    TableIdentifier tableIdentifier = TableIdentifier.of(RESTUtil.decodeNamespace(namespace), view);
-    if (icebergCatalogWrapperManager.getOps(prefix).existView(tableIdentifier)) {
-      return IcebergRestUtils.noContent();
+    String catalogName = IcebergRestUtils.getCatalogName(prefix);
+    Namespace icebergNS = RESTUtil.decodeNamespace(namespace);
+    LOG.info(
+        "Check Iceberg view exists, catalog: {}, namespace: {}, table: {}",
+        catalogName,
+        icebergNS,
+        view);
+    IcebergRequestContext context = new IcebergRequestContext(httpServletRequest(), catalogName);
+    TableIdentifier viewIdentifier = TableIdentifier.of(icebergNS, view);
+    boolean exists = viewOperationDispatcher.viewExists(context, viewIdentifier);
+    if (exists) {
+      return IcebergRestUtils.okWithoutContent();
     } else {
       return IcebergRestUtils.notExists();
     }
   }
+
+  @POST
+  @Path("{view}/metrics")
+  @Produces(MediaType.APPLICATION_JSON)
+  @Timed(name = "report-view-metrics." + MetricNames.HTTP_PROCESS_DURATION, absolute = true)
+  @ResponseMetered(name = "report-table-metrics", absolute = true)
+  public Response reportTableMetrics(
```

Comment:
Gravitino Iceberg REST server follows Iceberg REST API spec,  there are table metrics interfaces, seems no view metrics .  https://github.com/apache/iceberg/blob/main/open-api/rest-catalog-open-api.yaml#L1262

Reply:
Remove it!